### PR TITLE
feat(install): arm64 support

### PR DESCRIPTION
**Pull request by yarysp**
This PR brings the ARM64 arch build official !

Please test it a lot on Apple M1 :wink: 

I've also improved the docker generation and the install script based on the @vncloudsco, @4NK1T & @sa7mon PR proposed on reNgine
https://github.com/yogeshojha/rengine/pull/1007
https://github.com/yogeshojha/rengine/pull/1212
https://github.com/yogeshojha/rengine/pull/1210

### CODE REVIEW

**Comment by AnonymousWP** [comment]
I think you need to rebase this one due to the recent merge of https://github.com/Security-Tools-Alliance/rengine-ng/pull/81.

**Comment by yarysp** [comment]
> I think you need to rebase this one due to the recent merge of #81.

Yes but release/2.1.0 needs to be rebased first

File: web/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+ARG go_version=1.21.5
+ARG go_amd=go${go_version}.linux-amd64.tar.gz
+ARG go_arm=go${go_version}.linux-arm64.tar.gz
 
 # Labels and Credits
 LABEL \
-    name="reNgine" \
-    author="Yogesh Ojha <yogesh.ojha11@gmail.com>" \
-    description="reNgine is a automated pipeline of recon process, useful for information gathering during web application penetration testing."
+    name="reNgine NG" \
```
Comment (AnonymousWP) [inline web/Dockerfile:14]:
```suggestion
    name="reNgine-ng" \
```

File: web/Dockerfile [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+    author="Security-Tools-Alliance (https://github.com/Security-Tools-Alliance) & Yogesh Ojha <yogesh.ojha11@gmail.com>" \
+    description="reNGine NG is a automated pipeline of recon process, useful for information gathering during web application penetration testing."
```
Comment (AnonymousWP) [inline web/Dockerfile:16]:
```suggestion
    description="reNgine-ng is a automated pipeline of recon process, useful for information gathering during web application penetration testing."
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Pull request merged**